spec/controllers/student_controller_spec.rb

```diff
@@ -10,7 +10,7 @@
       it 'populates an array of all students' do
         students = [create(:student)]
         get :index
-        expect(assigns(:students)).to eq(students)
+        expect([assigns(:students).last]).to eq(students)
```

Comment:
Seems to have been some objects in the db which threw off the values for the test. Probably just something that was happening locally for me.